plugin/vcs/gitlab/sql_review.go

```diff
@@ -0,0 +1,56 @@
+package gitlab
+
+import (
+	_ "embed"
+
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	// CIFilePath is the local path for GitLab ci file.
+	CIFilePath = ".gitlab-ci.yml"
+	// SQLReviewCIFilePath is the local path for SQL review CI in GitLab repo.
+	SQLReviewCIFilePath = ".gitlab/sql-review.yml"
+	// SQLReviewScriptFilePath is the local path for SQL review script in GitLab repo.
+	SQLReviewScriptFilePath = ".gitlab/sql-review.sh"
+)
+
+// SQLReviewCI is the GitLab CI for SQL review in VCS workflow.
+//
+//go:embed .gitlab/sql-review.yml
+var SQLReviewCI string
+
+// SQLReviewScript is the SQL review script used in SQL review CI.
+//
+//go:embed .gitlab/sql-review.sh
+var SQLReviewScript string
+
+// SetupSQLReviewCI will update the GitLab CI content to add or update the SQL review CI.
+func SetupSQLReviewCI(gitlabCI map[string]interface{}, sqlReviewEndpoint string) (string, error) {
+	if gitlabCI["sql-review"] == nil {
+		// Add include for SQL review CI
+		var includeList []interface{}
+		// Docs for GitLab include syntax: https://docs.gitlab.com/ee/ci/yaml/includes.html
+		switch include := gitlabCI["include"].(type) {
+		case []interface{}:
+			includeList = append(includeList, include...)
+		case string, interface{}:
+			includeList = append(includeList, include)
+		}
+
+		includeList = append(includeList, map[string]string{"local": SQLReviewCIFilePath})
+		gitlabCI["include"] = includeList
+	}
+
+	// Add SQL review endpoint.
+	gitlabCI["sql-review"] = make(map[string]interface{})
```

Comment:
Consider making gitlabCI a struct than over-using interface.

Reply:
Seems we cannot use a specific struct for the ci content, as we don’t know what the content is in the users' repo, and also, many fields can have different types of value, like the "include" can be a string, a slice of string, map[string]interface{}, event slice of string and map

Reply:
But we can define the struct for SQL review CI config. Updated now.